Busybox find does not support -printf. Moreover, there are no symlinks
in the tarball, so this exclude is quite useless.

--- a/pygments.rb.gemspec
+++ b/pygments.rb.gemspec
@@ -21,6 +21,5 @@
   # s.extensions = ['ext/extconf.rb']
   s.require_paths = ['lib']
 
-  exclude = `find . -type l -printf '%P\\0'`.split("\0").map {|f| "':!#{f}'" } * ' '
   s.files = `git ls-files -- . #{exclude}`.split("\n")
 end
